splice_machine/src/main/java/com/splicemachine/SpliceKryoRegistry.java

```diff
@@ -942,5 +942,6 @@ public Optional read(Kryo kryo, Input input, Class<Optional> type) {
         instance.register(ImmutableList.class, 339);
         instance.register(TriggerInfo2.class,EXTERNALIZABLE_SERIALIZER,340);
         instance.register(DatabaseDescriptor.class,EXTERNALIZABLE_SERIALIZER,341);
+        instance.register(MergeOperation.class,EXTERNALIZABLE_SERIALIZER,342);
```

Comment:
Does this operation need to be registered in SpliceSparkKryoRegistry?

Reply:
i think yes, if we want to execute this on spark, right?

Reply:
If so, please register